docs/components/modeler/web-modeler/git-sync.md

```diff
@@ -0,0 +1,85 @@
+---
+id: git-sync
+title: Git sync
+description: Sync your Git repositories with Web Modeler.
```

Comment:
Not blocking by any means, but this might flag Bing as too short -- I would detail. For example, "Sync your Git repositories with Web Modeler to achieve X, Y, and Z."